clang-tools-extra/docs/clang-tidy/checks/readability/qualified-auto.rst

```diff
@@ -96,3 +96,23 @@ Note in the LLVM alias, the default value is `false`.
   matched against only the type name (i.e. ``Type``). E.g. to suppress reports
   for ``std::array`` iterators use `std::array<.*>::(const_)?iterator` string.
   The default is an empty string.
+
+.. option:: RespectOpaqueTypes
+
+  If set to `false` the check will use the canonical type to determine the type that ``auto`` is deduced to.
```

Comment:
`canonical type` is a term that users will not likely understand. Please use something like `underlying type`

Reply:
Please follow 80 characters limit.

Reply:
There was attempt to create documentation linter on Phabricator, but it's necessary to perform some "archeology" to find it.